src/sketch.lisp

```diff
@@ -34,6 +34,10 @@
    (%window :initform nil :accessor sketch-%window :initarg :window)
    (%delayed-init-funs :initform (make-array 0 :adjustable t :fill-pointer t)
                        :accessor sketch-%delayed-init-funs)
+   (%noise-seed :initform 7 :accessor sketch-%noise-seed)
+   (%noise-map :initform (make-hash-table) :accessor sketch-%noise-map)
+   (%noise-lod :initform 4 :accessor sketch-%noise-lod)
+   (%noise-falloff :initform 0.5 :accessor sketch-%noise-falloff)
```

Comment:
Maybe these (except for`%noise-map`) should be exported and have an `:initarg`, so that they could be easily passed from outside?

Reply:
Although making those available in the sketch's binding list won't work perfectly -- accessors would need to be `sketch-noise-seed`, `sketch-noise-lod` and `sketch-noise-falloff`, which is not ideal. Although it is possible rewriting `noise-detail` using `slot-value` and adding an `:after` method to the setters `(setf sketch-noise-seed)`, `(setf sketch-noise-lod)` and `(setf sketch-noise-falloff)`...

Probably best left as it is for now.